packages/application/src/application/resolvers/audit-outcome-resolver.ts

```diff
@@ -3,15 +3,17 @@ import { AuditOutcome } from '@src/infrastructure/repositories/issue-details';
 
 @injectable()
 export class AuditOutcomeResolver {
-  resolve(prevDatacap?: number | '', currentDatacap?: number | ''): AuditOutcome {
-    if (!prevDatacap || !currentDatacap) return AuditOutcome.UNKNOWN;
+  resolve(auditPrevDatacap?: number | string, auditCurrentDatacap?: number | string): AuditOutcome {
+    const prevDatacapAmount = Number(auditPrevDatacap);
+    const currentDatacapAmount = Number(auditCurrentDatacap);
+    if (!prevDatacapAmount || !currentDatacapAmount) return AuditOutcome.UNKNOWN;
 
     switch (true) {
-      case prevDatacap === currentDatacap:
+      case prevDatacapAmount === currentDatacapAmount:
         return AuditOutcome.MATCH;
-      case prevDatacap && prevDatacap * 2 === currentDatacap:
+      case prevDatacapAmount && prevDatacapAmount * 2 === currentDatacapAmount:
         return AuditOutcome.DOUBLE;
-      case prevDatacap && prevDatacap / 2 === currentDatacap:
+      case prevDatacapAmount && prevDatacapAmount / 2 === currentDatacapAmount:
```

Comment:
Is it possible in here that we'll run into resolution issues when (eg) 2.5 turns to 1.25 when expressed as full bytes amount? Like 1 byte off the === exact match?

Also we're missing REJECTED here - is that because rejected applications don't make it here?